apps/extension/src/routes/page/onboarding/set-grpc-endpoint.tsx

```diff
@@ -0,0 +1,34 @@
+import { Card, CardDescription, CardHeader, CardTitle } from '@penumbra-zone/ui/components/ui/card';
+import { FadeTransition } from '@penumbra-zone/ui/components/ui/fade-transition';
+import { usePageNav } from '../../../utils/navigate';
+import { PagePath } from '../paths';
+import { ServicesMessage } from '@penumbra-zone/types/src/services';
+import { GrpcEndpointForm } from '../../../shared/components/grpc-endpoint-form';
+
+export const SetGrpcEndpoint = () => {
```

Comment:
This file used to be `set-rpc-endpoint.tsx`. When I renamed it to `set-grpc-endpoint.tsx`, _and_ extracted the `GrpcEndpointForm` component, the file was so different that Git thought of it as an entirely different file.